### PR TITLE
Test fixes for compute_resource docker

### PR DESCRIPTION
**Pull request by adarshdubey-star**
This PR fixes the test_positive_upload_image failing because the container was not initiated to create an image from it.

### CODE REVIEW

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_docker.py

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_docker.py

**Review by shubhamsg199 — APPROVED** [review]
Ack, one suggestion

File: tests/foreman/cli/test_computeresource_docker.py [hunk 1]
```diff
@@ -71,19 +71,20 @@ def test_positive_upload_image(module_org, target_sat, container_contenthost):
         })
         Docker.container.start({'id': container['id']})
         """
-        container = {'uuid': 'stubbed test'}
+        container = {'uuid': 'test'}
         repo_name = gen_string('alphanumeric').lower()
 
         # Commit a new docker image and verify image was created
         image_name = f'{repo_name}/{CONTAINER_UPSTREAM_NAME}'
+        container_contenthost.execute('podman run -d -it --name test library/busybox')
```
Comment (shubhamsg199) [inline tests/foreman/cli/test_computeresource_docker.py:79]:
```suggestion
        container_contenthost.execute(f'podman run -d -it --name test {CONTAINER_UPSTREAM_NAME}')
```

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_docker.py

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_docker.py

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_docker.py

**Comment by devendra104** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_docker.py

File: tests/foreman/cli/test_computeresource_docker.py [hunk 2]
```diff
@@ -71,19 +71,22 @@ def test_positive_upload_image(module_org, target_sat, container_contenthost):
         })
         Docker.container.start({'id': container['id']})
         """
-        container = {'uuid': 'stubbed test'}
+        container = {'uuid': 'test'}
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_computeresource_docker.py:74]:
```suggestion
        container = {'name': gen_string('alphanumeric')}
```

File: tests/foreman/cli/test_computeresource_docker.py [hunk 3]
```diff
@@ -71,19 +71,22 @@ def test_positive_upload_image(module_org, target_sat, container_contenthost):
...
         repo_name = gen_string('alphanumeric').lower()
 
         # Commit a new docker image and verify image was created
         image_name = f'{repo_name}/{CONTAINER_UPSTREAM_NAME}'
+        container_contenthost.execute(
+            f'podman run -d -it --name test docker.io/{CONTAINER_UPSTREAM_NAME}'
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_computeresource_docker.py:80]: I think better to just specify only container name, and let podman decide where it wants to pull image from, wdyt?
```suggestion
            f'podman run -d -it --name {container["uuid"]} {CONTAINER_UPSTREAM_NAME}'
```

**Review by Gauravtalreja1 — COMMENTED** [review]
Ack, with a suggestion

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_computeresource_docker.py

**Comment by adarshdubey-star** [comment]
Handling https://github.com/SatelliteQE/robottelo/pull/12121 here.

**Comment by adarshdubey-star** [comment]
Handling this here https://github.com/SatelliteQE/robottelo/pull/12121 closing this one.

**Pull request closed without merging**